utils/parse.ts

```diff
@@ -0,0 +1,10 @@
+import { wei, WeiSource } from '@synthetixio/wei';
+
+export function parseSafeWei(v: WeiSource, def: WeiSource) {
```

Comment:
I'm pretty sure the exact same function somewhere else, above. Are you sure there are no duplicates?

Reply:
ive never seen it in this codebase, its always a direct conversion (NaN if invalid), can you send info on where else you have seen?